### ISSUE TITLE
Issue when parsing label

### ISSUE THREAD

**Issue by MikHulk** [brandly/elm-dot-lang#18]
Hi,

I try to use elm-dot-lang from my app on dot file generated from [pyreverse](https://pylint.pycqa.org/en/latest/pyreverse.html).

This tool generates this kind of dot file:
```dot
Digraph "classes" {
rankdir=BT
charset="utf-8"
"A" [color="black", fontcolor="black", label="{A|attr_name : NoneType\lname : NoneType\l}", shape="record", style="solid"];
"B" [color="black", fontcolor="black", label="B", shape="record", style="solid"];
"B" -> "A" [arrowhead="empty", arrowtail="none"];
}
```

Sadly when I try to parse this text with [fromString](https://package.elm-lang.org/packages/brandly/elm-dot-lang/latest/DotLang#fromString) I get this errors from parser:

```
0 = {
col = 71
problem = Expecting "n"
row = 4
}
1 = {
col = 71
problem = Expecting "t"
row = 4
}
2 = {
col = 71
problem = Expecting "r"
row = 4
}
3 = {
col = 71
problem = Expecting "\""
row = 4
}
4 = {
col = 71
problem = Expecting "\\"
row = 4
}
5 = {
col = 71
problem = Expecting "u{"
row = 4
}
```

I don't know if `\l` is authorized in dot language, I am digging into documentation.

**Comment by MikHulk** [comment]
I have found the meaning of this escaping: https://www.graphviz.org/doc/info/shapes.html#record.

I don't know how it's hard to do that. But if it is possible to parse record based nodes it would be great. I am looking into source to see that.

If you have any insight about where to look or fix something let me know.

**Comment by brandly** [comment]
Hey, sorry I didn't see this sooner! I don't have any insights yet, but I wanted to let you know that I've now seen this issue and I'll look at it more sometime soon. Based on that graphviz link, it looks like this is legitimate syntax that we should be able to parse.

I saw that you have a branch with a failing test. You could open a pull request with that if you'd like just to bring more visibility to it! Thanks for raising this issue.

**Comment by MikHulk** [comment]
Hi, 

Thank you for the reply. I plan to work on this issue on my spare time indeed. No problem to open a PR with the UT.

That said I have just started to work with parser in Elm. I have written a few very simple ones. So I think I will not go very fast on this. So please let me know if you find a solution before I do.

this is my PR: https://github.com/brandly/elm-dot-lang/pull/19

**Comment by brandly** [comment]
fixed by #21
